cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+
+Having said this, every baby is different so whilst some can get away with
+very little absorbency, others are considered 'medium to heavy wetters' and
+will need a lot more absorbency, relying much more heavily on hemp inserts
+(typically used in pocket diapers) or boosters (typically layered into other
+types of diapers to increase absorbency).
+
+### Absorbency speed
+This is important for two reasons:
+1. You need to layer your absorbency so that the layers closer to the baby
+are the ones that absorb liquids the fastest - this will allow that layer to
+absorb the liquids and wick them away to the lower layers which will have more
+storage capacity.
+2. If you have a baby who 'floods'; i.e. releases a large quantity of liquid
+in a short time frame. Here you want the fastest possible absorption to wick
+away the moisture so that you don't get leaks.
+
+## Newborn vs One Size
+There are 3 'sizes' of cloth diapers:
++ **Newborn**: These are extra small diapers that in some cases (not all) also
+fit preemies. They have snaps that allow them to be resized to 2 or 3 different
+sizes as well.
++ **One Size:** Often referred to as ***OS*** diapers, these are the typical
+size of modern cloth diaper that **will accompany a child throughout the
+majority of their diaper journey**. They are marketed as fitting babies from
+around 3-15kg (with a small amount of variation per brand) but usually don't fit
+newborns because the leg holes are designed for slightly bigger babies.
++ **XL:** Nomenclature varies but these are often designated 'XL' or 'Super'
+and are made for older toddlers but sizes vary widely. They could refer to
+diapers that are 9 - 18kgs for example or even diapers that are 15 - 20kgs, so
+it's important to review the stipulations of fit from the individual brand.
+
+**One size (OS) are by _far_ the most common**, with many people deciding to
+wait until their babies fit into this size before they start their cloth diaper
+journey.
+
+There _are_ 'newborn hacks' of folds that allow you to use OS diapers on smaller
+babies but these are greatly dependent on the baby's body and on the diaper
+brands.
+
+# Our Choices
+
+Our choices have evolved over time, not only as we have gained experience
+but also as little A has grown and had different requirements.
+
+## Diaper Sizes
+I chose to invest in a small stash of newborn cloth diapers because I wanted
+to start as soon as possible. I wanted to learn the systems and understand
+what we preferred to use _before_ buying a full stash of OS diapers that we
+might hate because they were the wrong type or materials.
+
+Unless you don't plan to start cloth diapering until baby is 3 months old or
+so, I think this is a good investment because it lets you learn the ropes and
+they hold their value extremely well (more so than OS) so can be sold on easily.
+
+
+## Diaper Types
+Now that we have 'dialed things in' a little more, there are three types of
```

Comment:
Either update the spelling "dialed" to "dialled"; ref: https://www.oxfordlearnersdictionaries.com/definition/english/dial-in

_or_ consider removing expression completely as it's not English-as-second-language friendly 
and doesn't add anything to the section.

```suggestion
There are three types of diapers we use.
```